.github/workflows/check-vulns.yml

```
@@ -46,16 +66,21 @@ jobs:
           cat result.log
       - name: build matrix
         id: set_matrix
         if: ${{ failure() }}
         working-directory: ./dep_checker
         run: |
-          matrix=$(grep -o '{.*}' result.log | jq -c .)
-          echo "matrix=$matrix"
+          # Extract vulnerabilities JSON from the log
+          vulnerabilities_json=$(grep -o '{.*}' result.log | tail -1)
+          echo "Raw vulnerabilities JSON: $vulnerabilities_json"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)